### PR TITLE
Change/remove backwards compatibility

### PR DESCRIPTION
**Pull request by bartvanb**
Addresses a number of points from #454

### CODE REVIEW

File: vantage6-client/vantage6/client/__init__.py [hunk 1]
```diff
@@ -612,24 +612,20 @@ def __init__(self, parent) -> None:
 class UserClient(ClientBase):
     """User interface to the vantage6-server"""
 
-    def __init__(self, *args, verbose=False, log_level='debug',
-                 **kwargs) -> None:
+    def __init__(self, *args, log_level='debug', **kwargs) -> None:
```
Comment (frankcorneliusmartin) [inline vantage6-client/vantage6/client/__init__.py:615]: Should the default level be `info`?

File: vantage6-server/vantage6/server/resource/collaboration.py
Comment (frankcorneliusmartin) [inline vantage6-server/vantage6/server/resource/collaboration.py:1]: Could you elaborate on why these are compatibility endpoints?

**Review by frankcorneliusmartin — COMMENTED** [review]
Ok

File: vantage6-server/vantage6/server/resource/collaboration.py
Comment (bartvanb) [reply vantage6-server/vantage6/server/resource/collaboration.py:1]: Not sure I understand the question, but e.g. endpoint `collaboration/<id>/nodes` is superseded by `/node?collaboration_id=<id>`

File: vantage6-client/vantage6/client/__init__.py (hunk 1 above)
Comment (bartvanb) [reply vantage6-client/vantage6/client/__init__.py:615]: We can make it `info` but I think we'd need to go over the log statements and check if some debugs should not be info level. This was I guess the easy choice, and also the same as we make by default for the node/server logs

**Comment by bartvanb** [comment]
We need to think about what to do with the permission checks of the (herein removed) double endpoints. Should we add the collaboration scope now in order to deal with that in a structured manner? Or keep them after all?

File: vantage6-client/vantage6/client/__init__.py (hunk 1 above)
Comment (frankcorneliusmartin) [reply vantage6-client/vantage6/client/__init__.py:615]: Ok lets leave it like this for now. I created an issue to keep track of this: https://github.com/vantage6/vantage6/issues/692

**Comment by frankcorneliusmartin** [comment]
> We need to think about what to do with the permission checks of the (herein removed) double endpoints. Should we add the collaboration scope now in order to deal with that in a structured manner? Or keep them after all?

I dont think that belongs within this PR. Lets work on that separately in https://github.com/vantage6/vantage6/issues/245

**Comment by bartvanb** [comment]
The collaboration scope, with the permission checks discussed above, is now under review in #711. So we can also proceed with merging this (after conflicts are resolved ofc)

**Comment by bartvanb** [comment]
Closing this, superseded by #741

**Pull request closed without merging**